Sources/Datadog/Tracing/TracingV2Configuration.swift

```diff
@@ -69,14 +72,7 @@ internal struct TracingMessageReceiver: FeatureMessageReceiver {
     ///
     /// - Parameter context: The updated core context.
     private func update(context: DatadogContext) -> Bool {
-        guard
-            let tracer = Global.sharedTracer as? Tracer,
-            let integration = tracer.rumIntegration
-        else {
-            return false
-        }
-
-        integration.attribues = context.featuresAttributes["rum"]?.all()
+        rum.attribues = context.featuresAttributes["rum"]?.all()
```

Comment:
This was causing TSAN data race crash, because following was possible:
* tracing feature registered in core
* → core dispatching new context
* at the moment of context arrival to this `TracingMessageReceiver`, the `tracer.rumIntegration` was in the middle of it's `alloc()`, causing data race on `tracer.rumIntegration` access.

The fix is to reverse the communication. Because `TracingMessageReceiver` is ready upon Tracing feature registration, the `Tracer` can access it safely.